runners/wikiEclipse/templates/createFile.asciidoc

```diff
@@ -0,0 +1,18 @@
+== Create the file <%= fileName; %>
+
+=== Creating <%= fileName; %> in the Eclipse IDE
+
+Create <%= fileName; %> in the Eclipse IDE<% if(contentFile) { %> and insert some data from <%= contentFile; %> into it.
+
+First, you want to open <%= contentFile; %> to get the data we want to put into <%= fileName; %>.
```

Comment:
The variable `contentFile` contains the path to the file with the content you want to insert, right? But when the user goes through a wiki tutorials, he doesn`t have the file, or am I thinking wrong right now?
So he can't open the file. I think we have to get the content from the file and then paste the whole content in the template.
Same for the other templates

Reply:
I can implement another case 
1. The file is in a subdirectory of the working directory and the user can get the content himself like I implemented it now 
2. The file is outside the working directory and the user has to copy the content of the file from the wiki and insert it into the file.
Or use the second case for every file, but then you have some ugly and long commands. What is your opinion?

Reply:
The user has no working directory in this sense. The wiki tutorial created is made available to the user in some form and he only has this description, no other files.  So he can't just open the other file to get the content. The content of the file must be displayed within the wiki. Then he can just copy and paste it.
It's the same way as in katacoda. The content is displayed in the tutorial text on the left-hand side and can then be copied from there into the editor.

Reply:
I think the key aspect is, that the tutorial must be understandable without having to do the tutorial actively. So all content that was changed from one file to another must be visible and comprehensible.The content file is usually not available to the user as file, that's why we need to print the content.